src/Datadog.Trace.ClrProfiler.Native/integration.h

```diff
@@ -157,6 +157,16 @@ struct MethodSignature {
     return 0;
   }
 
+  bool RetTypeIsObject() const {
```

Comment:
Nit: Can we call this `ReturnTypeIsObject()`? It's more readable than the abbreviation and more consistent with `ReturnTypeIsValueTypeOrGeneric()` elsewhere.

Reply:
Sure thing! Implemented in d86bd34965a511bc3dbe77d2c8cb3aa7deb72254